lib/MPD/Bed/Raw.pm

```diff
@@ -60,6 +60,11 @@ sub BUILD {
       $self->Chr, $self->Start, $self->End );
     return $self->log( 'fatal', $msg );
   }
+  if ( $self->Size > 2000 ) {
```

Comment:
Want to make sure this is something Thomas or Dave approved. Not sure what the right number here is

Reply:
2000 was Thomas' suggestion.  Should have made that clear in OP.

Reply:
This is an arbitrary decision that is based on my experience. We could empirically decide the right answer using testing but I am not sure that would be worth while at this time.